Formula/poac.rb

```diff
@@ -49,6 +49,8 @@ def install
     system "cmake", "--install", "build"
 
     man1.install (buildpath/"src/etc/man/man1").children
```

Comment:
While we're here, let's simplify this a little
```suggestion
    man.install "src/etc/man/man1"
```

Reply:
Thanks!